### ISSUE TITLE
 Load screen never finishes

### ISSUE THREAD

**Issue by burner1024** [gemrb/gemrb#1736]
#### Bug description
BG2-classic. It's a modded game. Mostly non-english, if that's relevant.
Similar to [this](https://github.com/gemrb/gemrb/issues/1289#issuecomment-896322987).

```
[Python/ERROR]: Traceback (most recent call last):
[Python/ERROR]:   File "/opt/gemrb/share/gemrb/GUIScripts/bg2/LoadScreen.py", line 70, in EndLoadScreen
[Python/ERROR]:     TMessageTA.Append("[p][color=f1f28d]" + GemRB.GetString (HintStr) + "[/color][/p]\n")
[Python/ERROR]: AttributeError: 'NoneType' object has no attribute 'Append'
```

#### Steps to reproduce
1. Load game.
2. Some sounds play, but it's stuck on loading screen.

#### Expected behavior
No error, normal load.

#### Screenshots
![Captura desde 2022-11-25 17-03-51](https://user-images.githubusercontent.com/1279852/203957152-8f7720f8-c82d-4099-93c1-9d2d2687b5ac.png)

GemRB version (check as many as you know apply)
- [x] master as of this issue
- [ ] 0.9.1
- [ ] 0.9.0
- [ ] 0.8.8

Video Driver (check as many as you know apply)
- [ ] SDL1.2
- [ ] SDL2 built with `USE_OPENGL_BACKEND`
- [x] SDL2 without `USE_OPENGL_BACKEND`

**Comment by lynxlynxlynx** [comment]
Is that the first error in the log?

**Comment by bradallred** [comment]
yeah, the message window should have been setup earlier in MessageWindow.py. This indicates something happened early in `MessageWindow::OnLoad()`. Either an old bg2 override from a previous version, or the modding has changed window/control ids. The former can be fixed by deleting GUIScripts/bg2/MessageWindow.py.

**Comment by burner1024** [comment]
Just realized I can't reliably reproduce this, so I wiped everything and re-compiled. Seems to be working now, I'll add more details if meet this again.

**Comment by burner1024** [comment]
Actually, fix for #1735 fixed this, that's why I couldn't reproduce anymore.
